cerbero/build/relocatabletar.py

```diff
@@ -0,0 +1,59 @@
+#------------------------------------------------------------------------------
+# RelocatableTar* are classes that represent a .tar file
+# with relocation abilities
+#------------------------------------------------------------------------------
+
+import os
+from tarfile import TarFile
+from cerbero.utils import shell, is_text_file
+from cerbero.tools.osxrelocator import OSXRelocator
+
+class RelocatableTar(TarFile):
+#------------------------------------------------------------------------------
```

Comment:
Use correct python docs

Reply:
Sure. This file comes directly from master: https://github.com/fluendo/cerbero/blob/master/cerbero/build/relocatabletar.py
I did a small modification, though